src/parser.yy

```diff
@@ -60,6 +60,18 @@ void yyerror(bpftrace::Driver &driver, const char *s);
   LOR        "||"
   PLUS       "+"
   PLUSPLUS   "++"
+
+  LANDASSIGN  "&&="
```

Comment:
Are there any other languages that have `&&=` and `||=`?

Reply:
Good question.

C/C++ doesn't have it, apparently: https://www.tutorialspoint.com/Compound-Assignment-Operators-in-Cplusplus

Neither does Python: https://www.programiz.com/python-programming/operators#assignment_operators

Or Java: https://www.geeksforgeeks.org/compound-assignment-operators-java/

Or JavaScript: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Assignment_Operators

They all have `<<=` and `>>=`, so I guess it makes more sense remove `&&=`/`||=` and add compound shift operators.

Reply:
Side note: funny, I was writing some C++ yesterday and I was hoping &&= existed.